client/sources/common/interpreter.py

```diff
@@ -317,7 +341,18 @@ def _compare(self, expected, code):
             print('# but got')
             print('\n'.join('#     {}'.format(line)
                             for line in actual.output_lines()))
-            raise ConsoleException
+            if not self.show_all_cases:
+                raise ConsoleException
+            elif self.CASE_PREFIX in code:
+                self.cases_total += 1
+                print(RED_X, f"Test Case {self.cases_total} failed")
+                format.print_line('-')
+                print()
```

Comment:
there's a `print()` here, but not in the subsequent `else` case?

Reply:
fixed by adding to else